docs/csharp/language-reference/operators/bitwise-and-shift-operators.md

```diff
@@ -161,9 +163,9 @@ For the complete list of C# operators ordered by precedence level, see the [Oper
 
 ## Shift count of the shift operators
 
-Prior to C# 11, for the shift operators `<<` and `>>`, the type of the right-hand operand must be `int` or a type that has a [predefined implicit numeric conversion](../builtin-types/numeric-conversions.md#implicit-numeric-conversions) to `int`. This restriction is removed in C# 11.
+For the built-in shift operators `<<`, `>>`, and `>>>`, the type of the right-hand operand must be `int` or a type that has a [predefined implicit numeric conversion](../builtin-types/numeric-conversions.md#implicit-numeric-conversions) to `int`.
```

Comment:
As far as I understand, that restriction is removed only for overloaded shift operators. For example, using `long` for the right-hand operand results in error for the built-in shift operator:
![image](https://user-images.githubusercontent.com/15279990/179406207-952f1d35-3075-4686-8f05-9880089bf004.png)